echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/jira/JiraNotificationService.java

```diff
@@ -37,36 +40,88 @@
 import org.springframework.stereotype.Component;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import retrofit.RetrofitError;
+import retrofit.client.Response;
 
 @Component
 @ConditionalOnProperty("jira.enabled")
 public class JiraNotificationService implements NotificationService {
   private static final Logger LOGGER = LoggerFactory.getLogger(JiraNotificationService.class);
   private static final int MAX_RETRY = 3;
-  private static final long RETRY_BACKOFF = 3;
+  private static final long RETRY_BACKOFF = 100;
```

Comment:
A retry backoff of 3 milliseconds seemed absurdly low, setting this to 100ms.

Reply:
Debatable.

Reply:
No strong opinions, just don't follow the point of sleeping for 3 milliseconds (my reasoning is it may as well be zero in that case). 🤷‍♂️

Reply:
My original comment was a joke, sorry ... I suspect the original was a big mistake.

Honestly, because it's a linear backoff and _only_ 3 attempts, I'd probably do a couple seconds or something.

Reply:
Hah, I thought maybe you were joking but I totally couldn't tell 😂